File: data-structure/sorting-algorithm/selectionsort/sorting.c
void selection_sort(int vetor[], int n) {
    for(int i = 0; i < n; i++) {
        int menor = i;
        for(int j = i + 1; j < n; j++) {
            if(vetor[j] < vetor[menor])
            menor = j;
        }
        int aux = vetor[i];
        vetor[i] = vetor[menor];
        vetor[menor] = aux;
    }
}

File: data-structure/sorting-algorithm/selectionsort/sorting.py
def selection_sort(lista)
    tamanho = len(lista)
    for j in range(tamanho-1)
        min_index = j
        for i in range(j, tamanho):
            if lista[i] < lista[min_index]:
                min_index = i
        if lista[j] > lista[min_index]:
            aux = lista[j]
            lista[j] = lista[min_index]
            lista[min_index] = aux
pr